pttrack/models.py

```diff
@@ -318,6 +323,15 @@ def notes(self):
 
         return note_list
 
+    def workup_notes(self):
```

Comment:
Yeah I would not mind that. I don't think this function is even used anywhere because the needs_workup field takes care of it. Ill get rid of it for now and then start using that convention in the future.